### ISSUE TITLE
Criteria to select the best model

### ISSUE THREAD

**Issue by jmendozais** [yzcjtr/GeoNet#52]
I just checked the loss of GeoNet on training, and its seems that it has a high variance. _How did you selected your best model for evaluation on depth and camera motion tasks?_.

**Comment by yzcjtr** [comment]
Hi, you are right. Training on the KITTI dataset is painful for the loss of scene diversity. You can select the KITTI 2012 training set as a validation set.
